src/mockServiceWorker.js

```diff
@@ -27,6 +27,14 @@ self.addEventListener('message', async function (event) {
   const allClientIds = allClients.map((client) => client.id)
 
   switch (event.data) {
+    case 'PING': {
```

Comment:
I agree that these events could be renamed into something like `KEEPALIVE_REQUEST` and `KEEPALIVE_RESPONSE` for better readability.